include/swift/Frontend/DiagnosticVerifier.h

```diff
@@ -36,21 +36,52 @@ bool verifyDependencies(SourceManager &SM, ArrayRef<SourceFile *> SFs);
 // MARK: - DiagnosticVerifier
 struct ExpectedFixIt;
 
+/// A range of text targeted by a fix-it, in terms of line and column numbers.
+struct FixItRange {
+  static constexpr unsigned NoValue = ~0u;
+
+  unsigned StartLine, StartCol;
+  unsigned EndLine, EndCol;
```

Comment:
Do you know if there's a case where we might emit fix-its in different files than the diagnostics?

Reply:
None that I know of.